controllers/controllers.js

```diff
@@ -42,16 +42,37 @@ exports.getArticleByID = (req, res, next) => {
 exports.getCommentsByArticleID = (req, res, next) => {
     const {article_id} = req.params
     const promises = [selectArticleByID(article_id)]
-
+    
     Promise.all(promises).then(() => {
         selectCommentsByArticle(article_id).then((comments) => {
             res.status(200).send({comments})
         })
+        
+    }).catch((err) => {
+        next(err)
+    })
+    
+}
 
+exports.postCommentsByArticleID = (req, res, next) => {
+    const {article_id} = req.params
+    const promises = [selectArticleByID(article_id)]
```

Comment:
checking that the article exists is not necessary because the article_id is a foreign key on the comments table and if it doesn't exist then sql will throw an error